bound.go

```diff
@@ -32,6 +32,15 @@ func NewBoundFromPoints(corner, oppositeCorner *Point) *Bound {
 	return b
 }
 
+// NewBoundAroundPoint creates a new bound given a center point,
+// and a distance from the center point in meters
+func NewBoundAroundPoint(center *Point, distance float64) *Bound {
```

Comment:
Should be `NewGeoBoundAroundPoint` since it's a lat/lng thing. Also maybe rename distance to radius